src/lib/dialog/dialog.spec.ts

```diff
@@ -965,6 +973,73 @@ describe('MatDialog', () => {
       expect(container.hasAttribute('aria-labelledby')).toBe(false);
     }));
   });
+
+  describe('global dialog options', () => {
+    function createTestComponent(openConfig: MatDialogConfig, globalConfig: DialogGlobalOptions) {
+      // Reset the previously configured testing module to be able set new providers.
+      // The testing module has been initialized in the root describe group for the dialogs.
+      TestBed.resetTestingModule();
```

Comment:
You should call `overlayContainer.ngOnDestroy` before resetting the testing module, otherwise the tests will leak overlay containers into other tests.